Rakefile

```diff
@@ -155,3 +155,26 @@ namespace :release do
     end
   end
 end
+
+namespace :pkg do
+  desc 'Sync the root rust-toolchain version to all crates'
+  task :'rust_version:sync' do
+    rust_version = File.open('rust-toolchain').read.chomp
+    regexp = /^rust-version = "(.*)"$/
+    next_rust_version = "rust-version = \"#{rust_version}\""
+
+    failures = Dir.glob("#{__dir__}/{,*/}Cargo.toml").map do |file|
```

Comment:
the `release` namespace uses a `FileList` to pick a list of files relevant to the task. Can we be consistent and use that here?

I think it simplifies this expressions to:

```ruby
  link_check_files = FileList.new('**/Cargo.toml') do |f|
    f.exclude('node_modules/**/*')
    f.exclude('**/target/**/*')
    f.exclude('**/vendor/**/*')
    f.include('Cargo.toml')
  end
```

I find this to be a bit more readable than the glob expansion style.

Reply:
Can do!

Just curious, would you expect this rake task to go more than one sub directory deep? From what I can see so far, all `Cargo.toml` files are one directory deep at most (and one at the root level).

If it's only one level deep, then isn't it "safer" to just go single `*` expansion instead of recursive `**` expansion `FileList.new('*/Cargo.toml').include('Cargo.toml')` ?

Reply:
that sounds good to me.